### PR TITLE
Clean-ups in BiT-ResNet model documentation and edits to my author page

### PR DESCRIPTION
**Pull request by sayakpaul**
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms and Google's Privacy Policy at https://www.google.com/policies/privacy.

Please provide only one commit. You can continuously add changes to your latest commit using [`git commit --amend`](https://stackoverflow.com/q/179123) followed by `git push -f` to update your remote branch. Alternatively, [here](https://stackoverflow.com/q/5189560) are details on how to squash multiple commits into one commit.

We check modified Markdown files for validity using [this](https://github.com/tensorflow/tfhub.dev/blob/master/.github/workflows/contributions-validator.yml) GitHub Workflow. You can execute the same checks locally by passing the file paths of modified Markdown files (relative to the `assets/docs` directory) to validator.py e.g. `python3 ./tools/validator.py google/google.md google/models/albert_base/1.md ...`.

### CODE REVIEW

**Comment by sayakpaul** [comment]
@MorganR this should be fixed.

File: assets/docs/sayakpaul/collections/bit-resnet/1.md [hunk 1]
```diff
@@ -36,7 +36,11 @@ Models included in this collection have two variants: (1) off-the-shelf inferenc
 ## References
```
Comment (MorganR) [inline assets/docs/sayakpaul/collections/bit-resnet/1.md:36]: Please double check the references here to be sure they match the model names. It looks like the classification models should say `distill_bit_resnet50x1`, while the feature extractors are `distill_bit_r50x1`. Also note that the classification models have been linked twice while the feature extractors are missing.

File: assets/docs/sayakpaul/bit_resnet152x2_384_feature_extraction/1.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Module sayakpaul/bit_r152x2_384_feature_extraction/1
```
Comment (MorganR) [inline assets/docs/sayakpaul/bit_resnet152x2_384_feature_extraction/1.md:1]: And here

File: assets/docs/sayakpaul/bit_resnet152x2_224_feature_extraction/1.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Module sayakpaul/bit_r152x2_224_feature_extraction/1
```
Comment (MorganR) [inline assets/docs/sayakpaul/bit_resnet152x2_224_feature_extraction/1.md:1]: And here

**Review by MorganR — CHANGES_REQUESTED** [review]
Your new changes look good, but I'm commenting with a few other changes mainly to fix issues with the links in the collection.

**Comment by sayakpaul** [comment]
Thanks, @MorganR! I have addressed the comments. My bad regarding the mistakes introduced in the filenames of the feature extractor. Is it possible to cover this scenario from the validator script?

**Review by MorganR — APPROVED** [review]
Thank you for fixing the issues! Agreed on checking this in the validator. We've already filed an internal FR to address this.

**Pull request merged**